Index: util.cc
===================================================================
--- util.cc	(revision 29)
+++ util.cc	(working copy)
@@ -455,7 +455,7 @@
     char *inbuf = (char*)s, *outbuf = t; // const char* -> char*. Ugh.
     size_t inbytes = l, outbytes = l;
 
-    while (iconv(ic, &inbuf, &inbytes, &outbuf, &outbytes) ==
+    while (iconv(ic, (const char**)&inbuf, &inbytes, &outbuf, &outbytes) ==
 	   (size_t)-1)
       if (errno == E2BIG) {
 	/* try to enlarge the output buffer */
@@ -508,7 +508,7 @@
     char *inbuf = (char*)s, *outbuf = t; // const char* -> char*. Ugh.
     size_t inbytes = l, outbytes = l;
 
-    while (iconv(ic, &inbuf, &inbytes, &outbuf, &outbytes) ==
+    while (iconv(ic, (const char**)&inbuf, &inbytes, &outbuf, &outbytes) ==
 	   (size_t)-1)
       if (errno == E2BIG) {
 	/* try to enlarge the output buffer */
@@ -580,7 +580,7 @@
     char *inbuf = s; wchar_t *outbuf = t;
     size_t inbytes = l, outbytes = l*sizeof(wchar_t);
 
-    if (iconv(myic[1], &inbuf, &inbytes, (char**)&outbuf, &outbytes) ==
+    if (iconv(myic[1], (const char**)&inbuf, &inbytes, (char**)&outbuf, &outbytes) ==
 	(size_t)-1)
       return NULL;
     /* terminate the output string */
