--- win32dist/readme.rtf.orig	2010-11-04 22:49:14.000000000 -0500
+++ win32dist/readme.rtf	2010-11-04 22:59:41.000000000 -0500
@@ -1,9 +1,9 @@
 {\rtf1\ansi\ansicpg932\deff0\deftab709{\fonttbl{\f0\froman\fprq2\fcharset128 Times New Roman;}{\f1\froman\fprq2\fcharset0 Times New Roman;}}
 {\*\generator Msftedit 5.41.15.1515;}\viewkind4\uc1\pard\lang1033\b\f0\fs24 Important Information\b0\par
 \par
-\tab To start PlasmaClient, run PCLauncher.exe. This accepts your regular Myst Online username and password.\par
+\tab To start PlasmaClient, run PCLauncher. This accepts your regular Myst Online username and password.\par
 \par
-\tab There have been some reports that when PlasmaClient is loading, you should not click on another window or use alt-tab because it can freeze. For some people this works fine \endash  if you experience loading issues, try keeping PlasmaClient focused during the entire loading process.\par
+\tab There have been some reports that when PlasmaClient is loading, you should not click on another window or use Command-Tab because it can freeze. For some people this works fine \endash  if you experience loading issues, try keeping PlasmaClient focused during the entire loading process.\par
 \par
 \b Controls\par
 \par
@@ -70,7 +70,7 @@
 \par
 \b Graphics Settings\b0\par
 \par
-\tab PlasmaClient does not use the Myst Online graphics settings. To change your PlasmaClient settings, use the\f1  "Graphics" tab of PCLauncher.exe\f0 . This will not affect your Myst Online graphics settings.\par
+\tab PlasmaClient does not use the Myst Online graphics settings. To change your PlasmaClient settings, select "Preferences" from the PCLauncher menu. This will not affect your Myst Online graphics settings.\par
 \par
 \f1\tab To disable the framerate counter, you can use the command "/fps off". To turn it on again, use "/fps on"\f0\par
 }
